Stashr install fails

Installation fails with the following output:

```bash
> cargo install stashr
    Updating crates.io index
  Downloaded stashr v0.3.3
  Downloaded 1 crate (6.5 KB) in 1.32s
  Installing stashr v0.3.3
    Updating crates.io index
  Downloaded anstyle v1.0.4
  Downloaded utf8parse v0.2.1
  Downloaded strsim v0.10.0
  Downloaded clap_derive v4.4.7
  Downloaded anstream v0.6.4
  Downloaded unicode-ident v1.0.12
  Downloaded fs_extra v1.3.0
  Downloaded anyhow v1.0.75
  Downloaded anstyle-query v1.0.0
  Downloaded clap_lex v0.6.0
  Downloaded either v1.9.0
  Downloaded quote v1.0.33
  Downloaded heck v0.4.1
  Downloaded clap v4.4.7
  Downloaded proc-macro2 v1.0.69
  Downloaded anstyle-parse v0.2.2
  Downloaded clap_builder v4.4.7
  Downloaded itertools v0.11.0
  Downloaded colorchoice v1.0.0
  Downloaded syn v2.0.38
  Downloaded 20 crates (944.1 KB) in 1.02s
   Compiling proc-macro2 v1.0.69
   Compiling unicode-ident v1.0.12
   Compiling utf8parse v0.2.1
   Compiling anstyle-query v1.0.0
   Compiling colorchoice v1.0.0
   Compiling anstyle v1.0.4
   Compiling clap_lex v0.6.0
   Compiling anyhow v1.0.75
   Compiling anstyle-parse v0.2.2
   Compiling heck v0.4.1
   Compiling strsim v0.10.0
   Compiling either v1.9.0
   Compiling fs_extra v1.3.0
   Compiling anstream v0.6.4
   Compiling itertools v0.11.0
   Compiling clap_builder v4.4.7
   Compiling quote v1.0.33
   Compiling syn v2.0.38
   Compiling clap_derive v4.4.7
   Compiling clap v4.4.7
   Compiling stashr v0.3.3
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> /Users/gleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/stashr-0.3.3/src/main.rs:1:12
  |
1 | #![feature(str_split_remainder)]
  |            ^^^^^^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0554`.
error: could not compile `stashr` (bin "stashr") due to previous error
error: failed to compile `stashr v0.3.3`, intermediate artifacts can be found at `/var/folders/2g/nwb9zhbj4kb0_x6y0r2yx5hw0000gn/T/cargo-install8EEIuY`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

Evidently this happens when trying to install stashr on a stable release of cargo. The README should indicate this and possibly also provide instruction on how to proceed in case this happens.

--- a/README.md
+++ b/README.md
@@ -26,3 +26,14 @@ stashr
 cargo install stashr
 ```
 
+### Troubleshooting
+
+You need to be on the nightly release channel of rust to install stashr.
+If you are not, you might get the following error: ``error[E0554]: `#![feature]` may not be used on the stable release channel``.
+
+To proceed first install [rustup](https://rustup.rs/), then run the following two commands:
+
+```bash
+rustup install nightly
+cargo +nightly install stashr
+```

